src/Components/Search/Suggestions/SuggestionItem.tsx

```diff
@@ -98,7 +105,13 @@ const DefaultSuggestion: React.FC<SuggestionItemProps> = ({
   const matches = match(display, query)
   const parts = parse(display, matches)
   const partTags = parts.map(({ highlight, text }, index) =>
-    highlight ? <strong key={index}>{text}</strong> : text
+    highlight ? (
+      <strong key={index} style={{ color: "blue" }}>
```

Comment:
`<strong color="blue100">` tag is not working.  
I can replace it with <Text> but it's not simple like strong. Also, the alignments change. 
![image](https://user-images.githubusercontent.com/17421923/225005954-237858c6-930a-419c-ae15-48054dbd8e7d.png)

Reply:
we can create the new styled component. something like this

```javascript
const Highlight = styled.strong`
  color: ${themeGet("colors.blue100")};
`
...
const partTags = parts.map(({ highlight, text }, index) =>
  highlight ? <Highlight key={index}>{text}</Highlight> : text
)
```